metagraph/src/graph/representation/succinct/boss_merge.cpp

```diff
@@ -179,27 +179,23 @@ BOSS::Chunk* merge_blocks_to_chunk(const std::vector<const BOSS*> &graphs,
     if (verbose)
         print_bin_stats(bins);
 
-    std::vector<BOSS::Chunk*> blocks(bins.front().size() - 1, NULL);
```

Comment:
I guess you weren't using this vector before anyway? The only reference to it that I see is getting its size later.

Reply:
Yeah, this was redundant, I noticed this when was changing these lines.